### ISSUE TITLE
Register of devices

### ISSUE THREAD

**Issue by SergiyIvanovAtGTI** [Azure/azure-iot-sdk-csharp#416]
- **OS, version, SKU and CPU architecture used:** <VERSION> 
- **Application's .NET Target Framework :** <NETVERSION> 
- **Device:** <DEVICETYPE> 
- **SDK version used:** <VERSION> 

# Description of the issue:

Could you change interface, please, so it should be possible to register > 1000 devices in less than 1 minute? 

# Code sample exhibiting the issue:

# Console log of the issue:

**Comment by CIPop** [comment]
/cc @nberdy @alextolp 

@SergiyIvanovAtGTI you should be able to register multiple devices in parallel today. Each device registration is normally done within a few seconds. 
If you are seeing slower response times, please open a support ticket within your Azure subscription to further investigate.

Closing since there isn't sufficient information: please fill in the new issue template with the required details.

**Comment by yzhong94** [comment]
@SergiyIvanovAtGTI , thank you for your contribution to our open-sourced project! Please help us improve by filling out this 2-minute [customer satisfaction survey](https://www.surveymonkey.com/r/Q5PLZ3V)
